README.md

```diff
@@ -60,84 +60,70 @@ npm run coverage
 ## Usage
 
 ### Stand-alone module
-To follow...
+```js
+const Udaru = require('udaru')
```

Comment:
I think lowercase `U` would be better, otherwise it may seem it is a constructor